readyplayerme/rpm-unreal-avatar-creator
# [SDK-166] Remove the draft when the back button is clicked.

## [TICKETID](https://ready-player-me.monday.com/boards/2563815861/pulses/TICKETID)

## Description

-   Added functionality to call the draft deletion API when the back button is pressed.

## Changes

#### Added

-   Added functionality to call the draft deletion API when the back button is pressed.



## How to Test

-   Edit existing avatar and without saving, click the back button. Edit the same avatar again, there should not be leftovers from the previous edition.

## Checklist
